.gitignore

```diff
@@ -12,6 +12,7 @@ Thumbs.db
 *.iml
 *.ipr
 *.iws
+/out/
```

Comment:
IntelliJ writes there. Sure, I can make it a separate commit. R.

On Sat, 3 Oct 2015 4:14 am David Schlosnagle <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/palantir/atlasdb/pull/80#discussion_r41086058>:
>
> > @@ -12,6 +12,7 @@ Thumbs.db
> >  *.iml
> >  *.ipr
> >  *.iws
> > +/out/
>
> What is writing to here? Should this be separate change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/palantir/atlasdb/pull/80/files#r41086058>.
>